src/gui/EditWidgetIcons.cpp

```diff
@@ -138,18 +145,25 @@ void EditWidgetIcons::load(const Uuid& currentUuid, Database* database, const Ic
 
 void EditWidgetIcons::setUrl(const QString& url)
 {
+#ifdef WITH_XC_HTTP
     m_url = url;
     m_ui->faviconButton->setVisible(!url.isEmpty());
     resetFaviconDownload();
+#else
+    m_ui->faviconButton->setVisible(false);
+#endif
 }
 
 void EditWidgetIcons::downloadFavicon()
```

Comment:
It is a slot that is defined in the constructor. Cleaner to just remove the inside and offers the potential for an info warning in the future if we don't just hide the button.